internal/service/url_service_test.go

```diff
@@ -13,7 +13,16 @@ type MockURLRepository struct {
 	mock.Mock
 }
 
-// TODO add mock methods for save and get url
+// created two mock method for save and ger url address
+func (m *MockURLRepository) SaveURL(url models.URL) error {
+	args := m.Called(url)
+	return args.Error(1)
```

Comment:
why 1 (Error(1))

Reply:
after the save method, we return the second value Error(1)